### PR TITLE
TechDebt: Improve ccms test handling

### PR DESCRIPTION
**Pull request by colinbruce**
## What

This adds some git & bash magic to the circle ci job.
If your commits on a branch include changes to files that match the regex `/services/ccms/*.*` the standard circle unit test job will run the CCMS tests too.

The overnight job will still run to ensure nothing slips through, but this should increase confidence that CCMS changes are monitored without having to manually run jobs

> ### Things to note
> The first unit_test should have taken the reduced time
> The TEMP commit should run all the tests because I changed a file that matched the /service/ccms/. pattern
> 
> Once this has been approved, I'll remove the temp commit, it's only to prove the theory

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing: `bundle exec rake`
- Github should not be reporting conflicts; you should have recently run `git rebase master`.
- There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- The PR description should say what you changed and why, with a link to the JIRA story.
- You should have looked at the diff against master and ensured that nothing unexpected is included in your changes.
- You should have checked that the commit messages say why the change was made.

### CODE REVIEW

**Comment by colinbruce** [comment]
The first unit_test should have taken the reduced time
The TEMP commit should run _all_ the tests because I changed a file that matched the /service/ccms/*.* pattern

File: app/services/ccms/attribute_value_generator.rb [hunk 1]
```diff
@@ -20,6 +20,9 @@ module CCMS
   # method name without the prefix is called on the appropriate object in the options hash, e.g.
   # 'vehicle_registration_number'  will call the registration_number method on options[:vehicle] in order to get the
   # value to insert.
+  #
+  # add a comment to trick circle into running the CCMS tests
```
Comment (willc-work) [inline app/services/ccms/attribute_value_generator.rb:24]: can this comment be removed before merge?

Comment (colinbruce) [reply app/services/ccms/attribute_value_generator.rb:24]: Yes, I'll remove the whole commit 👍 
I just wanted to demonstrate the different test-run scenarios for peace of mind 😄

**Review by stephenrichards — APPROVED** [review]

**Review by willc-work — APPROVED** [review]
looks great

**Pull request merged**